docs/source/design.rst

```diff
@@ -34,6 +34,10 @@ as any errors will affect at most a single request. Though as we describe below
 only processing a single request at a time requires some assumptions about how
 applications are programmed.
 
+The sync worker does not support keep-alive - each connection is closed after
```

Comment:
I defer to you, since English is my first language. Your judgement is more helpful here.